cartridges/int_klaviyo_core/cartridge/scripts/klaviyo/utils.js

```diff
@@ -0,0 +1,247 @@
+'use strict';
+
+/* API Includes */
+var Logger = require('dw/system/Logger');
+var Site = require('dw/system/Site');
+var StringUtils = require('dw/util/StringUtils');
+
+/* Script Modules */
+var klaviyoServices = require('*/cartridge/scripts/klaviyo/services.js');
+
+// event name constants
+var EVENT_NAMES = {
+    'viewedProduct' : 'Viewed Product',
+    'viewedCategory' : 'Viewed Category',
+    'searchedSite' : 'Searched Site',
+    'addedToCart' : 'Added to Cart',
+    'startedCheckout' : 'Started Checkout',
+    'orderConfirmation' : 'Order Confirmation',
+    'placedOrder' : 'Placed Order',
```

Comment:
We shouldn't need placed order or ordered product correct?

Reply:
fair point, we can remove these as no code in the cartridge refers to them

Reply:
Done